test/getCreateOptions.test.js

```diff
@@ -205,14 +211,34 @@ test('getCreateOptions sets brand to "joyent" by default', function (t) {
 
     createOpts = getCreateOptions(req);
 
-    t.equal(createOpts.brand, 'joyent', 'default brand should be joyent');
+    t.equal(createOpts.brand, req.config.default_container_brand,
+        'default brand should be joyent');
```

Comment:
Maybe won't be "joyent" - so just say "check container brand is the the default" ?

Reply:
Done!